### DISCUSSION TITLE
Messages Reprocessing After Channel Restart

### DISCUSSION THREAD

**Discussion by nkovalevich** [nextgenhealthcare/connect#6233] [General]
REPOST FROM MIRTH FORUMS: https://forums.mirthproject.io/forum/mirth-connect/support/185263-messages-reprocessing-after-channel-restart

For each of the healthcare systems we support, we have an individual JMS listener which routes through transformers per message, and into a database writer into MS SQL.
Whenever we restart the Mirth server or the database writer channel, all messages begin reprocessing automatically, including old messages and they are being reentered into our patient database. The only way to prevent this is to remove all messages before restarting the channel or the server, but this has caused problems for us when the server has unexpected issues and it restarts unprompted. Has anyone else seen this behavior or have any idea about what could be causing this? I think I provided enough information, but let me know what else I can provide for this.

Mirth Connect Server 4.5.0

Built on January 17, 2024

Java version: 17.0.6

Windows Server 2019, hosted in Azure

**Comment by pacmano1** [comment]
Refer to your forums post please.
